src/config.c

```diff
@@ -3168,6 +3168,7 @@ standardConfig static_configs[] = {
     createIntConfig("databases", NULL, IMMUTABLE_CONFIG, 1, INT_MAX, server.dbnum, 16, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("port", NULL, MODIFIABLE_CONFIG, 0, 65535, server.port, 6379, INTEGER_CONFIG, NULL, updatePort), /* TCP port. */
     createIntConfig("io-threads", NULL, DEBUG_CONFIG | IMMUTABLE_CONFIG, 1, 128, server.io_threads_num, 1, INTEGER_CONFIG, NULL, NULL), /* Single threaded by default */
+    createIntConfig("prefetch-batch-max-size", NULL, MODIFIABLE_CONFIG, 0, 128, server.prefetch_batch_max_size, 16, INTEGER_CONFIG, NULL, NULL),
```

Comment:
i personally think this is far too low level to be made a user config (and p.s. i don't see it in redis.conf).
maybe we can completely drop it, and if not, let's make it a HIDDEN one

Reply:
I truly added it in `redis.conf` https://github.com/redis/redis/pull/14017/files#diff-c683e35ac8c539e4f6c402fc159fe0e57086e3e661dd9ac7e48cedaee2c79c14R1337

I agree it is a low level thing, i am not sure if memory prefetching can work well in all platform (with different cache size), but i tested on EC2 type of AMD, Intel, ARM, it shows enabling memory prefetching is better. so if you want to hide this config, i can do that